tests/test_analyze_common.py

```diff
@@ -3,10 +3,16 @@
 # Copyright (c) 2020 VMware, Inc. All Rights Reserved.
 # SPDX-License-Identifier: BSD-2-Clause
 
+from __future__ import absolute_import
```

Comment:
We only support Python 3.5 and higher. Do we need this for something else?

Reply:
No, sorry. I miss delete such line. I'll remove it and update the commit message.

Reply:
Done! :smile: